### PR TITLE
Added some custom talkactions

### PR DESCRIPTION
**Pull request by daanyx**
Algumas Talkactions simples porém úteis!

### CODE REVIEW

**Review by adrianomengue — APPROVED** [review]
Good job!

**Review by ghost — APPROVED** [review]
Good idea, thanks

**Comment by mattyx14** [comment]
Is for to otx jeje

File: data/scripts/talkactions/god/online.lua [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    local players = Game.getPlayers()
+    local onlineList = {}
+
+    for _, targetPlayer in ipairs(players) do
+        if hasAccess or not targetPlayer:isInGhostMode() then
+            table.insert(onlineList, ("%s [%d]"):format(targetPlayer:getName(), targetPlayer:getLevel()))
+        end
+    end
+
+    local playersOnline = #onlineList
```
Comment (Costallat) [inline data/scripts/talkactions/god/online.lua]:
```suggestion
    local playersOnlineList = #onlineList
```

File: data/scripts/talkactions/god/online.lua [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    player:sendTextMessage(MESSAGE_ATTENTION, ("%d players online!"):format(playersOnline))
+
+    for i = 1, playersOnline, maxPlayersPerMessage do
```
Comment (Costallat) [inline data/scripts/talkactions/god/online.lua]:
```suggestion
    for i = 1, playersOnlineList, maxPlayersPerMessage do
```

File: data/scripts/talkactions/god/online.lua [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        local j = math.min(i + maxPlayersPerMessage - 1, playersOnline)
```
Comment (Costallat) [inline data/scripts/talkactions/god/online.lua]:
```suggestion
        local j = math.min(i + maxPlayersPerMessage - 1, playersOnlineList)
```

File: data/scripts/talkactions/god/online.lua [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    local onlineList = {}
+
+    for _, targetPlayer in ipairs(players) do
+        if hasAccess or not targetPlayer:isInGhostMode() then
+            table.insert(onlineList, ("%s [%d]"):format(targetPlayer:getName(), targetPlayer:getLevel()))
+        end
+    end
+
+    local playersOnline = #onlineList
+    player:sendTextMessage(MESSAGE_ATTENTION, ("%d players online!"):format(playersOnline))
```
Comment (Costallat) [inline data/scripts/talkactions/god/online.lua]:
```suggestion
    player:sendTextMessage(MESSAGE_ATTENTION, ("%d players online!"):format(playersOnlineList))
```

File: data/scripts/talkactions/player/aol.lua [hunk 5]
```diff
@@ -0,0 +1,15 @@
+local aol = TalkAction("!aol")
+
+function aol.onSay(cid, words, param)
+local aol = 2173
```
Comment (Costallat) [inline data/scripts/talkactions/player/aol.lua]:
```suggestion
local aolID = 2173
```

File: data/scripts/talkactions/player/aol.lua [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+  if doPlayerRemoveMoney(cid, 50000) == true then
+    doPlayerSendTextMessage(cid,MESSAGE_EVENT_ADVANCE, "You have bought an aol for 50K!")
+    doPlayerAddItem(cid, aol, 1)
```
Comment (Costallat) [inline data/scripts/talkactions/player/aol.lua]:
```suggestion
    doPlayerAddItem(cid, aolID, 1)
```

**Review by dudantas — DISMISSED** [review]

**Review by dudantas — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Much better, avoids overload in the future

**Review by majestyotbr — APPROVED** [review]

**Pull request merged**